inclusive-design/idrc
# feat: use logotype on inner pages (resolves #62)

## Description

Shows logo with logotype on all pages except home.

TODO:

- [ ] Updated hover, focus, and active states for logo (brand)

## Steps to test

1. Visit home page and inner pages on deploy preview.

**Expected behavior:** Logo without logotype appears on home, logo with logotype appears everywhere else.

## Additional information

Not applicable.

## Related issues

Resolves #62.
